pkg/policy/execute.go

```diff
@@ -232,12 +232,14 @@ func (e *Executor) createView(ctx context.Context, v *View) error {
 
 func (e *Executor) ExecutePolicies(ctx context.Context, req *ExecuteRequest, policies Policies, selector []string) (*ExecutionResult, error) {
 	var rest []string
+	pnames := make([]string, len(policies))
 	if len(selector) > 0 {
 		rest = selector[1:]
 	}
 	var found bool
 	total := ExecutionResult{PolicyName: req.Policy.Name, Passed: true, Results: make([]*QueryResult, 0)}
 	for _, p := range policies {
+		pnames = append(pnames, p.Name)
```

Comment:
```suggestion
		pnames[i] = p.Name
```
`i` being the `for` index. Otherwise you're just doubling the slice and first elements are zero values